storybook/src/components/tabs/tabs.core-template.ts

```diff
@@ -18,7 +20,13 @@ export const coreTabs: ComponentImplementation<Tabs<TemplateResult>> = {
                 href=${ifDefined(href)}
                 ?active=${modifier === "active"}
                 ?disabled=${modifier === "disabled"}
-                @dsoTabSwitch=${dsoTabSwitch}
+                @dsoTabSwitch=${(e: DsoTabCustomEvent<TabsSwitchEvent>) => {
+                  if (href && !e.detail.isModifiedEvent) {
+                    e.detail.originalEvent.preventDefault();
```

Comment:
Ik dacht dat we de afnemer zouden laten beslissen over het al dan niet volgen van de link